src/changelog/unreleased.md

```diff
@@ -44,9 +44,14 @@ changelog entry.
 
 - On Web, let events wake up event loop immediately when using
   `ControlFlow::Poll`.
+- On macOS, set the window theme on the `NSWindow` instead of application-wide.
```

Comment:
I'd consider that this is a _fix_, given that it was not behaving as on any other platform, though, it's a change in behavior which could be excluded from the point release... Not sure about it.